### PR TITLE
Importers gh6470

### PR DESCRIPTION
**Pull request by dezzie**
changes in this PR:
- specify in upload error message the valid file types
- if the error message specifies that the user is contacting support, change the link to go to support section instead of documentation section.

the original (stale) issue is here: https://github.com/Automattic/wp-calypso/issues/6470

### CODE REVIEW

File: client/my-sites/importer/error-pane.jsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ class ImporterError extends React.PureComponent {
 	contactSupport = event => {
 		event.preventDefault();
 		event.stopPropagation();
-		Page( '/help' );
+		Page( '/help/contact' );
```
Comment (andfinally) [inline client/my-sites/importer/error-pane.jsx]: I'm in two minds about the usefulness of jumping straight to a contact form vs going to the support page about this, e.g. https://support.wordpress.com/import/#importing-content-from-another-wordpress-blog. (This redirects me to the page for my language). Would you say a contact form is more helpful?

That page didn't mention zips! I've updated it - good thing you had another look at this! 👍

File: client/my-sites/importer/error-pane.jsx [hunk 2]
```diff
@@ -51,11 +51,13 @@ class ImporterError extends React.PureComponent {
 	};
 
 	getUploadError = () => {
-		const defaultError = this.props.translate( 'Unexpected error during the upload' );
+		const defaultError = this.props.translate(
+			'Oops! We ran into an unexpected error while uploading your file.'
```
Comment (andfinally) [inline client/my-sites/importer/error-pane.jsx]: This is a nice improvement on the previous wording!

File: client/my-sites/importer/error-pane.jsx [hunk 3]
```diff
@@ -51,11 +51,13 @@ class ImporterError extends React.PureComponent {
...
+		);
 		const { description = '' } = this.props;
 
 		return this.props.translate(
-			'%(errorDescription)s{{br/}}Try another file or {{cs}}contact support{{/cs}}.',
+			'%(errorDescription)s{{br/}}Make sure you are using a valid WordPress export file in XML or ZIP format. If that does not work, {{cs}}contact Support{{/cs}}.',
```
Comment (andfinally) [inline client/my-sites/importer/error-pane.jsx]: I'd suggest using contractions to make the tone more friendly, i.e.

```
'%(errorDescription)s{{br/}}Make sure you\'re using a valid WordPress export file in XML or ZIP format. If that doesn\'t work, {{cs}}contact Support{{/cs}}.',
```

**Comment by mattsherman** [comment]
@dezzie Would be nice to have the details of the proposed fix right in the description of this PR, instead of requiring navigation to a separate issue, especially when that issue seems to have morphed from its original issue (the description seems out of date).

File: client/my-sites/importer/error-pane.jsx (hunk 1 above)
Comment (mattsherman) [reply client/my-sites/importer/error-pane.jsx]: I agree -- I feel like we should send customers to the documentation first, unless there is no documentation that could be useful. Sending customers directly to support seems prone to increase the load on support.

@andfinally Thanks for updating the support doc to mention zips!

Comment (dezzie) [reply client/my-sites/importer/error-pane.jsx]: I don't think the increased load on support would be significant given the traffic to importers, but I'm fine keeping the link as is. I will still change the original language then to refer to Help section, not support.

**Comment by dezzie** [comment]
Sounds good @mattsherman, I added the description here, but to address the suggestions above the new PR is here: https://github.com/Automattic/wp-calypso/pull/33971

**Pull request closed without merging**